### ISSUE TITLE
Component to wrap text nodes

### ISSUE THREAD

**Issue by SamyPesse** [brigand/react-safe-html#6]
Currently all html nodes can be rendered, but it's not possible to wrap text nodes.

For example:

``` js
const components = {
    '#text': ({children} => <span>{children}</span>
};

<ReactSafeHtml html={html} components={components} />
```

`#text` can default to `({children} => React.Children.only(children)`

**Comment by brigand** [comment]
Good idea! Published as 0.4.0. Docs are at the bottom of the readme.

**Comment by SamyPesse** [comment]
Awesome! Thank you @brigand 🍻
